src/Components/Components/src/Rendering/NullDispatcher.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components.Rendering
+{
+    internal class NullDispatcher : Dispatcher
```

Comment:
null-object pattern for dispatcher since it's possible to get in a component now.